lexicons/com/atproto/identity/resolveIdentity.json

```diff
@@ -0,0 +1,38 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.resolveIdentity",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Resolves an identity (DID or Handle) to a full identity (DID document and verified handle).",
+      "parameters": {
+        "type": "params",
+        "required": ["identifier"],
+        "properties": {
+          "handle": {
```

Comment:
`handle` to something more generic that could be either? e.g. `resolveIdentity?id={did or handle}`

Reply:
also good catch! "identifier"